src/dds_manager.cpp

```diff
@@ -965,10 +994,18 @@ bool DDSManager::replaceFilter(const std::string& topicName,
                 continue;
             }
 
-            m_domainParticipant->delete_contentfilteredtopic(topicDesc);
+            // Now delte the content filter
```

Comment:
```suggestion
            // Now delete the content filter
```
(or remove the line, if it's already clear from the code what's happening)